t/t2201-job-cmd.t

```diff
@@ -229,6 +229,14 @@ test_expect_success 'flux-job: raise fails with invalid jobid' '
 	test_must_fail flux job raise foo
 '
 
+test_expect_success 'flux-job: raise fails with invalid jobids' '
+	test_must_fail flux job raise foo fee
+'
+
+test_expect_success 'flux-job: raise fails if note set twice' '
+	test_must_fail flux job raise --message=hi ${validjob} -- hello
+'
+
```

Comment:
not necessarily related to this PR, but AFAICT (maybe I'm not seeing it) we don't have any tests that ensure the message is actually posted in the eventlog correctly.  Perhaps should add some while we're at it.

Or perhaps I'm not seeing them, I did a great for "flux job cancel" and didn't see an obvious place they were tested.

Reply:
You're right. I'll open an issue on this one.